specification/ws.extensions/x-kaazing-ping-pong/src/main/scripts/org/kaazing/specification/ws.extensions/x-kaazing-ping-pong/should.not.escape.binary.frame/response.rpt

```diff
@@ -0,0 +1,36 @@
+accept http://localhost:8001/echo
+accepted
+connected
+
+read method "GET"
+read version "HTTP/1.1"
+read header "Host" "localhost:8001"
+read header "Upgrade" /(?i:websocket)/
+read header "Connection" /(?i:Upgrade)/
+read header "Sec-WebSocket-Key" /(?<key>[a-zA-Z0-9+\/=]{24})/
+read header "Sec-WebSocket-Version" "13"
+read header "Sec-WebSocket-Extensions" "x-kaazing-ping-pong"
+
+write status "101" "Switching Protocols"
+write version "HTTP/1.1"
+write header "Upgrade" "websocket"
+write header "Connection" "Upgrade"
+write header "Sec-WebSocket-Accept" ${ws:handshakeHash(key)}
+write header "Sec-WebSocket-Extensions" "x-kaazing-ping-pong; 01010102"
+
+# Expect a binary message
+read [0x82 0x87] ([0..4] :readMask)
+read option mask ${readMask}
+read [0x01 0x01 0x01 0x02]
+read [0x41 0x42 0x43]
+read option mask [0x00 0x00 0x00 0x00]
+
+# Send echoed response, starting with the escape frame
+write [0x82 0x07 0x01 0x01 0x01 0x02]
+
```

Comment:
Thank you! I think I've added an older version of the script on the repository, because I know I corrected this error and the test passed on my machine...